### ISSUE TITLE
allow host to be omitted in Swagger back-end

### ISSUE THREAD

**Issue by abstratt** [sdaschner/jaxrs-analyzer#122]
In Swagger, the host is optional. If I want to generate a Swagger spec that can relative to the host serving it, you just omit the host property. Also, host is optional. Right now, if a deployment domain is not specified, it defaults to "" in SwaggerBackend, which is an invalid valid for the property. It should just be omitted instead,

**Comment by abstratt** [comment]
Similar/same as #42.

**Comment by sdaschner** [comment]
Yes, that should do the trick... Would you like to help out with this feature and create a PR? :-)

**Comment by abstratt** [comment]
Also, should alliw for explicit API nsme and version. Currently, it uses the maven srtifact id and version.

**Comment by abstratt** [comment]
I have been doing this work as a contractor for Genologics/Illumina. Changes are in forks under their organization. Take a look and let me know if they seem good, and I can create a pull-request.

**Comment by sdaschner** [comment]
Sorry for the late response. Yes looks good to me, thanks!
